index.html

```diff
@@ -410,7 +410,9 @@ <h3>FOSSASIA | Projects</h3><br>
     {% for projects in site.data.orgprojects %}
     <div class="row v-align-children extra_padding_b">
       <div class="col-md-7 col-sm-6 text-center mb-xs-24">
+        <a href="{{projects.url}}">
         <img class="cast-shadow" alt="Screenshot" src="img/{{projects.img}}" />
```

Comment:
Indent the line `<img class="cast-shadow" alt="Screenshot" src="img/{{projects.img}}" />`

Reply:
done